doc/Manuals/LoadBalancing/ig-lb_x-road_external_load_balancer_installation_guide.md

```diff
@@ -0,0 +1,738 @@
+- [5. Configuring data replication with rsync over SSH](#5-configuring-data-replication-with-rsync-over-ssh)
+  * [5.1 Set up SSH between slaves and the master](#51-set-up-ssh-between-slaves-and-the-master)
+  * [5.2 Set up periodic configuration synchronization](#52-set-up-periodic-configuration-synchronization)
+    + [5.2.1 RHEL: Use `systemd` for configuration synchronization](#521-rhel-use-systemd-for-configuration-synchronization)
+    + [5.2.2 Ubuntu: Use upstart and cron for configuration synchronization](#522-ubuntu-use-upstart-and-cron-for-configuration-synchronization)
+    + [5.3 Set up log rotation for the sync log](#53-set-up-log-rotation-for-the-sync-log)
+
+<!-- tocstop -->
+
+## License
+
+This document is licensed under the Creative Commons Attribution-ShareAlike 3.0 Unported License. To view a copy of this
+license, visit http://creativecommons.org/licenses/by-sa/3.0/.
+
+## 1. Introduction
+
+### 1.1 Target Audience
+
+The intended audience of this installation guide are the X-Road security server administrators responsible for installing
+and configuring X-Road security servers to use external load balancing. The document is intended for readers with a good
+knowledge of Linux server management, computer networks, database administration, clustered environments and the X-Road
+functioning principles.
+
+### 1.2 References
+
+| Document Id    |  Document                                                                                |
+|:--------------:|:-----------------------------------------------------------------------------------------|
+| \[SS-CLUSTER\] | [Readme: Security server cluster setup with Ansible](../../../ansible/ss_cluster/README.md) |
+| \[IG-SS\] | [X-Road: Security Server Installation Guide](../ig-ss_x-road_v6_security_server_installation_guide.md) |
+| \[UG-SS\] | [X-Road 6 Security Server User Guide](../) |
+
+
+## 2. Overview
+
+This document describes the external load balancing support features implemented by X-Road and the steps necessary to
+configure security servers to run as a cluster where each node has an identical configuration, including their keys and
+certificates. X-Road security server configuration changes are handled by a single master server and one or more slave
+servers.
+
+### 2.1 Goals and assumptions
+
+The primary goal of the load balancing support is, as the name suggests, load balancing, not fault tolerance.
+A clustered environment increases fault tolerance but some X-Road messages can still be lost if a security server node fails.
+
+The implementation does not include a load balancer component. It should be possible to use any external load balancer
+component that supports HTTP-based health checks for the nodes and load balancing at the TCP level (eg. haproxy, nginx,
+AWS ELB or Classic Load Balancing, or a hardware appliance). A health check service is provided for monitoring a node's
```

Comment:
Should this read "AWS ALB or Classic Load Balancer", since ALB and Classic Load Balancer are the two types of AWS ELB's?